extensions/intellij/intellij_generator_plugin/src/test/resources/generator/RxGeneratedBlocListGenerator/sample_list_bloc.rxb.g.dart

```diff
@@ -0,0 +1,68 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
```

Comment:
How this corresponds to the alchemist change ?

Reply:
The file was missing, which caused a test to fail. I have restored it to resolve the issue.